### ISSUE TITLE
PTN001060286 tetraspanins "cell surface receptor signaling pathway"

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-annotation#2244]
PTN001060286 tetraspanins "cell surface receptor signaling pathway" is only supported by one annotation that I do not think is suitable for propagation.

This family of proteins is not well-characterised, but they are generally not thought of as directly involved in cell surface signaling. They might act as scaffolds in intracellular trafficking, regulate signaling, protein maturatio, membrane microdomain organization.....http://jcs.biologists.org/content/127/17/3641.....

**Comment by ValWood** [comment]
I agree.  Tetraspanins or claudins are membrane fusion protein more generally.
https://www.ncbi.nlm.nih.gov/pubmed/29134248

**Comment by marcfeuermann** [comment]
PTHR19282 is a huge family. It is the kind of family which is not well characterized but for which there is a lot of literature (126 different BP annotations in total, mostly phenotypes). Many mammalian members have been associated with receptor protein maturation and complex association, receptor internalization and regulation of various signaling pathways. However, I agree that they seem not to be DIRECTLY involved in cell surface signaling and I've removed the propagation of "cell surface receptor signaling pathway". I have updated the famly and added some annotations restricted to specific branches.
